### DISCUSSION TITLE
Highlights for ESMValCore v2.11.0 release

### DISCUSSION THREAD

**Discussion by chrisbillowsMO** [ESMValGroup/ESMValCore#2412] [General]
@ESMValGroup/esmvaltool-developmentteam This discussion collects possible highlights for ESMValCore v2.11.0 release. 

Please add pull requests that you want to nominate below and they will be highlighted in the upcoming release!

Only [already merged PRs](https://github.com/ESMValGroup/ESMValCore/pulls?q=is%3Apr+merged%3A%3E2023-12-19+-label%3Acondalock+base%3Amain+sort%3Acreated-asc|) are allowed, please. 😁

**Comment by bouweandela** [comment]
Maybe one of the highlights could be the many performance improvements in the preprocessor that we've worked on.

Another one could be that we have several new preprocessor functions

**Comment by valeriupredoi** [comment]
Here's my 2 pennies:

highlight no. 1: all @bouweandela did
highlight no. 2: all @schlunma did
highlight no. 3: MO is doing the release

:grin: :beer:

**Comment by axel-lauer** [comment]
Here is my list of highlights:
#2258 (new preprocessor local_solar_time)
#2299 (new preprocessors histogram and distance_metrics)
#2120 (performance improvement for preprocessor extract_levels)

**Reply by LisaBock** [reply]
I support Axel's suggestions.

**Reply by valeriupredoi** [reply]
I too am with The Axel :beer:

**Comment by valeriupredoi** [comment]
on a serious note now, I think it's a good idea to mention (somewhere in the first lines of the Changelog) that this is MO's first release - a very cool thing to have, of which we hope to have many more in the future :grin:
